packages/eslint-plugin/utils/constants.js

```diff
@@ -36,7 +36,8 @@ const TRANSLATION_FUNCTIONS = new Set( [ '__', '_x', '_n', '_nx' ] );
  *
  * @type {RegExp}
  */
-const REGEXP_SPRINTF_PLACEHOLDER = /%(((\d+)\$)|(\(([$_a-zA-Z][$_a-zA-Z0-9]*)\)))?[ +0#-]*\d*(\.(\d+|\*))?(ll|[lhqL])?([cduxXefgsp%])/g;
+const REGEXP_SPRINTF_PLACEHOLDER =
```

Comment:
The regex explainer comments alignment below can be fixed in a follow up PR

Reply:
Fixed in an extra commit, thanks for spotting this :+1: